src/components/layout.js

```diff
@@ -1,27 +1,28 @@
 import React from "react"
 import styled from "styled-components"
+import "normalize.css";
 
+import { Box } from "grommet";
 class Layout extends React.Component {
   render() {
     const {  children } = this.props
   
     return (
         <Wrapper>
-          <div
-            style={{
-              marginLeft: `auto`,
-              marginRight: `auto`,
-              maxWidth: '900px',
-            }}
-          >
-            <header/>
-            <main>{children}</main>
-          </div>
-          <Footer>
-            © {new Date().getFullYear()}, Built with
-            {` `}
-            <a href="https://www.gatsbyjs.org">Gatsby</a>
-          </Footer>
+          <Box background={{
+            "size": "small",
+            "position": "absolute",
+            "image": "url(./assets/backgroundHeader.svg)"}}> 
+            
+              <Box direction="row" justify="center" align="center" width={{"min": "350", "max": "1400"}}>
```

Comment:
For some reason the max width of the container is much wider than 1400 px 😬:
![image](https://user-images.githubusercontent.com/12502321/82296117-60305080-99b9-11ea-9ae2-07601a414f18.png)